doc/requirements.rst

```diff
@@ -118,8 +113,7 @@ the "async for" protocol implemented in Python 3.5.0 and 3.5.1. Motor continues
 to work with "async for" loops in Python 3.5.2 and later.
 
 Motor 1.2.5 and 1.3.1 add compatibility with Python 3.7, but at the cost of
-dropping Python 3.4.3 and older. Python 3.4.4 and later are supported by all
-Motor versions.
+dropping Python 3.4.3 and older.
```

Comment:
Let's add a line saying that Motor 2.2 dropped support for python 2.7 and 3.4.

Reply:
Done.